TimeSeriesAnalysis.m

```diff
@@ -120,7 +131,7 @@
 
 pkg load statistics
 
-% Formatstring (change this if the input files get different fields. %* means that the field is ignored in textread.
+% Formatstring (change this if the input files get different fields.

```

Comment:
Ignore commit. Jeg vil gerne have forklaringen for %* bliver stående.

Reply:
Det er ændret nu :-)

Reply:
Jeg har ændret det til 
% Formatstring (change this if the input files get different fields). The '%*' part in front of some of the formats means that the field is ignored in textread.